packages/client/test/rpc/engine/newPayloadV4.spec.ts

```diff
@@ -50,6 +50,8 @@ describe(`${method}: call with executionPayloadV4`, () => {
     const { pragueJson, pragueTime } = readyPragueGenesis(genesisJSON)
     const { service, server } = await setupChain(pragueJson, 'post-merge', { engine: true })
     const rpc = getRpcClient(server)
+    // const res1 = await rpc.request(`eth_getBlockByNumber`, ["0x0",false])
```

Comment:
Should this be commented out or can we remove it?

Reply:
i want to retain it to get the genisis data to build out the test if/when something changes which causes genesis change like what happened here.

Reply:
uncommented and added a new assert